xliff/lib.en-US.xlf

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2">
+  <file original="lib.json" source-language="en-US" target-language="en-US" datatype="plaintext">
+    <body>
+      <trans-unit id="loc.messages.LIB_UnhandledEx">
+        <source>Unhandled: %s</source>
+        <target state="final">Unhandled: %s</target>
+      </trans-unit>
+      <trans-unit id="loc.messages.LIB_FailOnCode">
+        <source>Failure return code: %d</source>
+        <target state="final">Failure return code: %d</target>
+      </trans-unit>
+    </body>
+  </file>
+</xliff>
```

Comment:
[ 'en-US', 'de-DE', 'es-ES', 'fr-FR', 'it-IT', 'ja-JP', 'ko-KR', 'ru-RU', 'zh-CN', 'zh-TW' ]